docs/user/filesystem.md

```diff
@@ -195,29 +195,283 @@ The contents of multiple pools/containers will be accessible via this
+$  mkdir /scratch_fs/daos_dfuse_samir
+$  mount.fuse3 dfuse /scratch_fs/daos_dfuse_samir -o pool=samirrav_pool,container=samirrav_cont
+$  touch /scratch_fs/daos_dfuse_samir/foo
+$  ls -l /scratch_fs/daos_dfuse_samir/
+total 0
+-rw-rw-r-- 1 samirrav samirrav 0 Sep 23 15:49 foo
+$  df -h | grep fuse
+dfuse                         537G  5.1G  532G   1% /scratch_fs/daos_dfuse_samir
+$
+```
+
+#### 2> Use dfuse entry in /etc/fstab to mount the container.
+
+```bash
+$  dmg pool create --scm-size=8G --nvme-size=64G --label=admin_pool
+Creating DAOS pool with manual per-engine storage allocation: 8.0 GB SCM, 64 GB NVMe (12.50% ratio)
+Pool created with 11.11%,88.89% storage tier ratio
+--------------------------------------------------
+  UUID                 : 97196853-a487-41b2-a5d2-286e62f14e9e
+  Service Ranks        : [1-5]
+  Storage Ranks        : [0-7]
+  Total Size           : 576 GB
+  Storage tier 0 (SCM) : 64 GB (8.0 GB / rank)
+  Storage tier 1 (NVMe): 512 GB (64 GB / rank)
+
+$  daos cont create admin_pool admin_cont --type=POSIX
+  Container UUID : ac4fb4db-a15e-45bf-8225-b71d34e3e578
+  Container Label: admin_cont
+  Container Type : POSIX
+
+Successfully created container ac4fb4db-a15e-45bf-8225-b71d34e3e578
+$  daos cont get-prop admin_pool admin_cont
+Properties for container admin_cont
+Name                                    Value
+----                                    -----
+Highest Allocated OID                   0
+Checksum                                off
+Checksum Chunk Size                     32 KiB
+Compression                             off
+Deduplication                           off
+Dedupe Threshold                        4.0 KiB
+EC Cell Size                            64 KiB
+Performance domain affinity level of EC 1
+Encryption                              off
+Global Version                          2
+Group                                   root@
+Label                                   admin_cont
+Layout Type                             POSIX (1)
+Layout Version                          1
+Max Snapshot                            0
+Owner                                   root@
+Redundancy Factor                       rf0
+Redundancy Level                        node (2)
+Performance domain affinity level of RP 3
+Server Checksumming                     off
+Health                                  HEALTHY
+Access Control List                     A::OWNER@:rwdtTaAo, A:G:GROUP@:rwtT
+$  echo 'dfuse /scratch_fs/root_dfuse fuse3 pool=admin_pool,container=admin_cont,auto,x-systemd.requires=daos_agent.service    0 0' >> /etc/fstab
+$  mkdir /scratch_fs/root_dfuse
+$  df -h | grep fuse
+$  mount -a
+$  df -h | grep fuse
+dfuse                         537G  5.1G  532G   1% /scratch_fs/root_dfuse
+$
+```
+
+#### 3> Use systemd service to mount the dfuse container.
+
+User can create the systemd file from /etc/fstab using the 'systemd-fstab-generator' command.
```

Comment:
Root again and not user, right?
There is a systemd for regular users that would be great to explore:
https://wiki.archlinux.org/title/Systemd/User

This would allow use to set up one dfuse instance per user logging on the system and is an alternative to multi-user dfuse.

Reply:
Added example on how user can use systemd to mount/unmount the dfuse.